### PR TITLE
[CP-1961] [File manager] Duplicates modal changes into upload failed on device reconnect

### PR DESCRIPTION
**Pull request by patryk-sierzega**
Jira: [CP-1961]

**Description**
- added resetting upload state :)

[CP-1961]: https://appnroll.atlassian.net/browse/CP-1961?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/files-manager/components/duplicated-files-modal/duplicated-files-modal.component.tsx [hunk 1]
```diff
@@ -78,7 +78,6 @@ const DuplicatedFilesListItemText = styled(Text)`
 
 const DuplicatedFilesModal: FunctionComponent = ({ ...props }) => {
   const filesManagerError = useSelector(getFilesManagerError)
-
```
Comment (OskarMichalkiewicz) [inline packages/app/src/files-manager/components/duplicated-files-modal/duplicated-files-modal.component.tsx:81]: What happened here : )?

Comment (patryk-sierzega) [reply packages/app/src/files-manager/components/duplicated-files-modal/duplicated-files-modal.component.tsx:81]: I think it looks better :D Do you think differently? :D

Comment (OskarMichalkiewicz) [reply packages/app/src/files-manager/components/duplicated-files-modal/duplicated-files-modal.component.tsx:81]: i just though that it was some IDE auto correction shenanigans, if you did this on purpose that seems okay : D

**Review by OskarMichalkiewicz — APPROVED** [review]

File: packages/app/src/device-manager/listeners/device-detached.listener.ts [hunk 2]
```diff
@@ -7,10 +7,10 @@ import { ipcRenderer } from "electron-better-ipc"
 import store from "App/__deprecated__/renderer/store"
 import { ListenerEvent } from "App/device-manager/constants"
 import { setConnectionStatus } from "App/device/actions"
+import { resetUploadingState } from "App/files-manager/actions"
 
-// AUTO DISABLED - fix me if you like :)
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-const deviceDetachedHandler = (_: any, _data: string): void => {
+const deviceDetachedHandler = (_: Electron.IpcRendererEvent): void => {
```
Comment (lkowalczyk87) [inline packages/app/src/device-manager/listeners/device-detached.listener.ts:12]: please consider detaching event for state reset

**Review by lkowalczyk87 — APPROVED** [review]

**Pull request merged**